en/docs/develop/customizations/adding-a-user-signup-workflow.md

```diff
@@ -2,240 +2,45 @@
 
 This section explains how to attach a custom workflow to the user signup operation in the API Manager. 
```

Comment:
shall I update this with following so that it gives an overall description?
'This section explains how to enable a simple approval workflow to the user signup operation in the API Manager. '

Reply:
added

Reply:
removed as requested